ReleaseProcessAutomation.Tests/Steps/Releases/ReleaseRcStepTests.cs

```diff
@@ -0,0 +1,231 @@
+    {
+      _gitClientStub.Setup(_ => _.IsWorkingDirectoryClean()).Returns(true);
+      _gitClientStub.Setup(_ => _.IsCommitHash("")).Returns(true);
+      _gitClientStub.Setup(_ => _.IsOnBranch("release/")).Returns(true);
+      _gitClientStub.Setup(_ => _.GetCurrentBranchName()).Returns("branchName");
+      _gitClientStub.Setup(_ => _.DoesBranchExist("prerelease/v0.0.0")).Returns(false);
+      var nextPossibleVersions = new SemanticVersion().GetNextPossibleVersionsDevelop();
+
+      _inputReaderMock.Setup(_ => _.ReadVersionChoice(It.IsAny<string>(), nextPossibleVersions)).Returns(new SemanticVersion()).Verifiable();
+
+      var rcStep = new ReleaseRCStep(
+          _gitClientStub.Object,
+          _config,
+          _inputReaderMock.Object,
+          _ancestorStub.Object,
+          _msBuildInvokerMock.Object,
+          _continueAlphaBetaMock.Object,
+          _consoleStub.Object);
+
+      Assert.That(
+          () => rcStep.Execute(new SemanticVersion(), "", false, false, "release/v1.3.5"),
+          Throws.Nothing);
+      _inputReaderMock.Verify();
+
+    }
+
+    [Test]
+    public void Execute_callsNextVersioFromHotfix_WithProperVersionCollection ()
+    {
+      _gitClientStub.Setup(_ => _.IsWorkingDirectoryClean()).Returns(true);
+      _gitClientStub.Setup(_ => _.IsCommitHash("")).Returns(true);
+      _gitClientStub.Setup(_ => _.IsOnBranch("release/")).Returns(true);
+      _gitClientStub.Setup(_ => _.GetCurrentBranchName()).Returns("branchName");
+      _gitClientStub.Setup(_ => _.DoesBranchExist("prerelease/v0.0.0")).Returns(false);
+      var nextPossibleVersions = new SemanticVersion().GetNextPossibleVersionsHotfix();
+
+      _inputReaderMock.Setup(_ => _.ReadVersionChoice(It.IsAny<string>(), nextPossibleVersions)).Returns(new SemanticVersion()).Verifiable();
+
+      var rcStep = new ReleaseRCStep(
+          _gitClientStub.Object,
+          _config,
+          _inputReaderMock.Object,
+          _ancestorStub.Object,
+          _msBuildInvokerMock.Object,
+          _continueAlphaBetaMock.Object,
+          _consoleStub.Object);
+
+      Assert.That(
+          () => rcStep.Execute(new SemanticVersion(), "", false, false, "hotfix/v1.3.5"),
+          Throws.Nothing);
+      _inputReaderMock.Verify();
+
+    }
+
+    [Test]
+    public void Execute_WithPauseForCommit_DoesNotCallNextStep ()
+    {
+      _gitClientStub.Setup(_ => _.IsWorkingDirectoryClean()).Returns(true);
+      _gitClientStub.Setup(_ => _.IsCommitHash("")).Returns(true);
+      _gitClientStub.Setup(_ => _.IsOnBranch("release/")).Returns(true);
+      _gitClientStub.Setup(_ => _.GetCurrentBranchName()).Returns("branchName");
+      _gitClientStub.Setup(_ => _.DoesBranchExist("prerelease/v0.0.0")).Returns(false);
+      var nextPossibleVersions = new SemanticVersion().GetNextPossibleVersionsHotfix();
+      _continueAlphaBetaMock.Setup(_ => _.Execute(It.IsAny<SemanticVersion>(), It.IsAny<string>(), It.IsAny<string>(), false)).Verifiable();
```

Comment:
this is no redundant sicne we already verify below. Do you want to use the AAA syntax in this file or stick to setup + verify. right now, you only did 1/2 of the change in this one test so I'm a bit confused.

Reply:
Oh no, this isnt the right file. There are some leftovers from renaming all the files, this is going to be an extra step when swapping to the new repo. Added it in the description: https://re-motion.atlassian.net/browse/RMRPA-6

Reply:
All ReleaseProcessAutomation.Tests.*** Files can be ignored